.ci-operator.yaml

```diff
@@ -0,0 +1,4 @@
+build_root_image:
+  name: release
+  namespace: openshift
+  tag: rhel-8-release-golang-1.21-openshift-4.16
```

Comment:
@stevekuznetsov should this be rhel 8 or 9, do you know?

Reply:
Unless there's a good reason, RHEL9 please. The automation will push us in that direction and getting ahead of the curve helps.

Reply:
@rashmigottipati please make the change to 9 - thanks!

Reply:
Updated to RHEL9